### ISSUE TITLE
Error when building ng prod

### ISSUE THREAD

**Issue by chdxD1** [TimMaa/project-bladerunner#16]
```
ERROR in ng:///usr/src/app/src/app/login/login.component.html (8,12): Property 'userService' is private and only accessible within class 'LoginComponent'.

ERROR in ng:///usr/src/app/src/app/login/login.component.html (16,15): Property 'userService' is private and only accessible within class 'LoginComponent'.

ERROR in ng:///usr/src/app/src/app/login/login.component.html (23,15): Property 'userService' is private and only accessible within class 'LoginComponent'.

ERROR in ng:///usr/src/app/src/app/login/login.component.html (2,1): Property 'userService' is private and only accessible within class 'LoginComponent'.

ERROR in ng:///usr/src/app/src/app/game/game.component.html (68,1): Property 'userService' is private and only accessible within class 'GameComponent'.
```

**Comment by TimMaa** [comment]
We are already onto this issue!

The error is inside the .ts-files.
Sometimes the linting is erroneous

**Comment by TimMaa** [comment]
@chdxD1 Please check if it was resolved with Merge #19
